litmus-portal/okteto-dev-env-setup/README.md

```diff
@@ -76,25 +70,25 @@ This directory contains setup guide to start developing Litmus Portal on Okteto
 
   Download the binary executable from [here](https://downloads.okteto.com/cli/okteto.exe) and add it to your `$PATH` environment variable.
 
-- <h4>STEP-6:</h4> Click the `Develop on Okteto` button on `env` folder's `README.md` from GitHub UI to deploy litmus-portal on Okteto cloud and start developing.
+- <h4>STEP-5:</h4> Click the `Develop on Okteto` button on `my-env` folder's `README.md` from GitHub UI to deploy litmus-portal on Okteto cloud and start developing.
   
-  [![Develop on Okteto](https://okteto.com/develop-okteto.svg)](https://cloud.okteto.com/deploy?repository=https://github.com/%GITHUB_USERNAME%/litmus&branch=dev)
+  [![Develop on Okteto](https://okteto.com/develop-okteto.svg)](https://cloud.okteto.com/deploy?repository=https://github.com/${GITHUB_USERNAME}/litmus&branch=dev)
 
-- <h4>STEP-7:</h4> Download and export the kubeconfig file from Okteto cloud settings page into `KUBECONFIG` environment variable and Login to Okteto cloud using the CLI.
+- <h4>STEP-6:</h4> Download and export the kubeconfig file from Okteto cloud settings page into `KUBECONFIG` environment variable and Login to Okteto cloud using the CLI.
 
   ```bash
   export KUBECONFIG=$HOME/Downloads/okteto-kube.config:${KUBECONFIG:-$HOME/.kube/config}
   okteto login
   ```
 
-- <h4>STEP-8:</h4> Go to specific component folders i.e. authentication, cluster-agents/subscriber, graphql-server, tools/self-deployer or frontend of `litmus-portal` folder and then run `okteto up` before manking changes to the code.
+- <h4>STEP-7:</h4> Go to specific component folders i.e. authentication, cluster-agents/subscriber, graphql-server, tools/self-deployer or frontend of `litmus-portal` folder and then run `okteto up` before manking changes to the code.
```

Comment:
since only `frontend` has an okteto.yml, maybe this should just mention that folder? (I can also contribute  okteto.yaml for the other components, if you think is useful)

Reply:
Yes, it will be great @rberrelleza, as of now the use case was just for the frontend, we may add Okteto as a dev environment for backend components too.